examples/ps/main.py

```diff
@@ -0,0 +1,20 @@
+from ollama import ps, pull, chat
+
+pull('mistral')
```

Comment:
nit: pull blocks without feedback since it doesn't stream which leaves the user wondering what's going on